packages/nc-gui-v2/components/smartsheet-column/EditOrAdd.vue

```diff
@@ -125,7 +134,12 @@ watchEffect(() => {
             <!-- Cancel -->
             {{ $t('general.cancel') }}
           </a-button>
-          <a-button html-type="submit" type="primary" size="small" @click="addOrUpdate(reloadMeta), (advancedOptions = false)">
+          <a-button
+            html-type="submit"
+            type="primary"
+            size="small"
+            @click="addOrUpdate(reloadMetaAndData), (advancedOptions = false)"
```

Comment:
This should give you a warning from Vue.
Not a good way to bind events, using comma operator.
Either add a new function to the component that handles the click or inline a function

```
@click="() => { ... }"
```